code/modules/events/_event.dm

```diff
@@ -31,8 +31,11 @@
 	var/auto_add = TRUE				//Auto add to the event pool, if not you have to do it yourself!
 	var/can_malf_fake_alert = FALSE	//Can be faked by malf ai?
 
-
-	var/dynamic_should_hijack = FALSE	// Whether or not dynamic should hijack this event
+	/// Whether or not dynamic is allowed to cancel these events if it is planning to run its own midround event soon,
+	/// or has run a midround event recently.
+	/// Avoid enabling this setting without adding a dynamic ruleset to replace it as it will significantly decrease
```

Comment:
this spacing upsets me greatly, please finish the sentence on the same line

Reply:
It will be too long for most people to read with standard editor sizes if the sentance is on 1 line. Comments should be broken into multiple lines if they are too long otherwise it is difficult to read.